tests/NIRCam_ecfs/S3_NIRCam.ecf

```diff
@@ -20,9 +20,12 @@ ff_outlier      True        # Set False to use only background region (recommend
 bg_thresh       [5,5]       # Double-iteration X-sigma threshold for outlier rejection along time axis
 
 # Background parameters
-bg_hw       12          # Half-width of exclusion region for BG subtraction (relative to source position)
-bg_deg      1           # Polynomial order for column-by-column background subtraction, -1 for median of entire frame
-p3thresh    5           # X-sigma threshold for outlier rejection during background subtraction
+bg_hw           12          # Half-width of exclusion region for BG subtraction (relative to source position)
+bg_deg          0           # Polynomial order for column-by-column background subtraction, -1 for median of entire frame
+p3thresh        5           # X-sigma threshold for outlier rejection during background subtraction
+bg_disp         False       # Row-by-row BG subtraction (only useful for NIRCam)
+bg_x1           None        # Left edge of exclusion region for row-by-row BG subtraction
+bg_x2           None        # Right edge of exclusion region for row-by-row BG subtraction
```

Comment:
Why not set bg_disp to True in the testing ECF to get some more testing coverage?